src/app/screens/connect/addressPurposeBox.tsx

```diff
@@ -0,0 +1,79 @@
+import { getTruncatedAddress } from '@utils/helper';
+import { AddressPurpose } from 'sats-connect';
+import styled from 'styled-components';
+
+const AddressBox = styled.div((props) => ({
+  padding: `${props.theme.spacing(10)}px ${props.theme.space.m}`,
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  backgroundColor: props.theme.colors.elevation6_800,
+  marginBottom: 1,
+  ':first-of-type': {
+    borderTopLeftRadius: props.theme.radius(2),
+    borderTopRightRadius: props.theme.radius(2),
+  },
+  ':last-of-type': {
+    borderBottomLeftRadius: props.theme.radius(2),
+    borderBottomRightRadius: props.theme.radius(2),
+  },
+}));
```

Comment:
my bad on the explicit height thing. I didn't see the stacks .bns names thing.

so I guess you want something like max-height: 60px
![Uploading image.png…]()